common/app/routes/Challenges/views/invalid/NotFound.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { PureComponent } from 'react';
+
+// import PropTypes from 'prop-types';
+/* eslint max-len: ["error", { "ignoreStrings": true }] */
+
+
+const propTypes = {};
+
+export class NotFound extends PureComponent {
```

Comment:
We already have a `NotFound` component [here](https://github.com/freeCodeCamp/freeCodeCamp/blob/92da7987d99fdf92f24e063fa512db52f30d5ccb/common/app/NotFound/Not-Found.jsx#L7). Please develop the existing component.

Reply:
Gotcha. I added the code to modify NotFound component